ops/terraform/services/eft/iam.tf

```
@@ -109,9 +112,38 @@ resource "aws_iam_policy" "eft_user" {
 }
 
 resource "aws_iam_role" "partner_bucket_role" {
-  for_each = local.eft_partners_config
+  for_each = local.eft_partners_config && var.enable_cloudtamer_deployment.enable_legacy_role_creation == true ? local.eft_partners_config : {}
```

```suggestion
  for_each = local.eft_partners_config ? local.eft_partners_config : {}
```

Make sure that _legacy_ (non-Kion/CT) Roles/resources/etc. are _always_ applied regardless of the CloudTamer "feature flag"--basically, there's no reason to have any variance here since the legacy roles are the default and we want to create them at all times.